src/discord/commands/moderation/nickname.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    Command, AccessLevel,
+    CommandError, Logger,
+    Guards
+} from "dd-botkit";
+
+export const NicknameCommand: Command = {
+    opts: {
+        name: "nickname",
+        access: AccessLevel.MODERATOR,
+        category: "Moderation",
+        guards: [
+            Guards.Argumented(
+                "nickname",
+                "Force changes the nickname of a user",
+                [{
+                    name: "user",
+                    type: "user",
+                    required: true
+                },
+                {
+                    name: "nickname",
+                    type: "string",
+                    required: true
+                }]
+            )
+        ]
+    },
+    handler: async (msg, next) => {
+        const [user, nickname] = msg.args;
+
+          await msg.guild.members.get((user as any).id).setNickname(nickname)
```

Comment:
You should probably check if the `ClientUser` has permission to change nickname. I am not sure if BotKit will handle that for you.

Reply:
BotKit does not track **bot** permissions whatsoever as that is out of the scope of the framework.

Reply:
@EricRabil I think @olvrb was talking about checking if the user who is typing has permissions

Reply:
Resolved, due to BotKit checking access level.